skyportal/models.py

```diff
@@ -585,6 +585,14 @@ class Obj(Base, ha.Point):
         doc="Notifications regarding the object sent out by users",
     )
 
+    listings = relationship(
```

Comment:
Obj.listings is never used anywhere, so we can omit this (& the associated back_populates reference)